src/foremast/templates/configs/configs.json.j2

```diff
@@ -17,6 +16,7 @@
         "app_grace_period": 0,
         "max_inst": 3,
         "min_inst": 1,
+        "ssh_keypair": null,
```

Comment:
Region may not be populated, by moving that default logic to the code, I was able to handle the specified and unspecified case. Lets talk.